iOSApp/Screens/Registration/RegistrationSecondViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  RegistrationSecondViewController.swift
+//  iOSApp
+//
+//  Created by Inas Ahmic on 20. 3. 2021..
+//
+
+import UIKit
+
+class RegistrationSecondViewController: UIViewController, UITextFieldDelegate {
+    
+    @IBOutlet weak var selected: UIImageView!
+    
+    @IBOutlet weak var notSelected: UIImageView!
+    
+    @IBOutlet weak var firstName: CustomTxtField!{
+        didSet{
+            firstName.delegate = self
+        }
+    }
+    
+    @IBOutlet weak var lastName: CustomTxtField!{
+        didSet{
+            lastName.delegate = self
+        }
+    }
+    
+    @IBOutlet weak var age: CustomTxtField!{
+        didSet{
+            age.delegate = self
+        }
+    }
+    
+    @IBAction func backToLogin(_ sender: UIButton) {
+        let storyboard = UIStoryboard(name: "Login", bundle: nil)
+        let viewcontoroller = storyboard.instantiateViewController(identifier: "LoginViewController")
```

Comment:
String has nice function that might help in this situation. 
Namely, you can use String(describing: LoginViewController.self) and have the same result.